test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -54,3 +55,19 @@ Feature: Status Desktop Chat
  #   Scenario: User cannot delete another user's message
  #        When user joins chat room test
  #        Then the user cannot delete the last message
+
+    Scenario Outline: The user can do a mention
+		When user joins chat room test
+		 And the user inputs a mention to <displayName> with message <message>
+        Then the <displayName> mention with message <message> have been sent
+    Examples:
+		| displayName | message          |
+          | tester123   |  testing mention |
+
+    Scenario Outline: The user can not do a mention to not existing users
+		When user joins chat room test
+		Then the user cannot input a mention to a non existing user <displayName>
```

Comment:
lol you actually had the right spelling here.

Reply:
:sweat_smile: